senza/templates/postgresapp.py

```diff
@@ -123,19 +125,39 @@
           Action: sts:AssumeRole
       Path: /
       Policies:
-      - PolicyName: AmazonEC2ReadOnlyAccess
+      - PolicyName: AmazonS3FullAccess
```

Comment:
I think the PolicyName is just a name and real access control is defined by the Effect, Action and Resource. You have a point though, maybe we can restrict the policy to a single bucket and/or EC2/EBS instances. The reason we need S3 access is for WAL-E logs and for the mint bucket, and at least WAL-E bucket should be writable. As for the EC2 and EBS, we only need to assign tags.